rails/app/services/api/v1/create_collaboration.rb

```diff
@@ -72,6 +72,16 @@ def json_result
       external_activity_url = add_param(external_activity_url, 'collaborators_data_url', collaborators_data_url)
 
       external_activity_url = add_param(external_activity_url, 'logging', @offering.clazz.logging)
+
+      # append authentication token info if needed
+      if @offering.runnable.append_auth_token
+        AccessGrant.prune!
+        token = @owner_learner.user.create_access_token_with_learner_valid_for(3.minutes, @owner_learner)
+        # add domain_uid first so we can pop the token off in the tests
+        external_activity_url = add_param(external_activity_url, 'domain_uid', @owner_learner.user.id)
```

Comment:
Can the domain_uid be added always even if `@offering.runnable.append_auth_token` is false?
This will be useful for SSO implementations in the future.

Also now that I see this code, I can see that the automated test in the AP of the collaboration launch should be doable. It just needs to get this json_result.

Reply:
Changed in https://github.com/concord-consortium/rigse/pull/1099/commits/093edb35f3a87ac72e0ed4c04fe7f78fc622ba92